multiversx/mx-chain-go
# codeHash in scdeploy and upgrade log

## Reasoning behind the pull request
- Extended the log event for scdeploy and upgrade
- Changed the version of the `mx-chain-vm-common-go` module
- Changed the version of the `mx-chain-es-indexer-go` module
  
## Proposed changes
- Extended the log event that is generated when a smart contract is deployed or upgraded, the log will contains an extra topic with the `codeHash`
- PR from `mx-chain-vm-common-go`: https://github.com/multiversx/mx-chain-vm-common-go/pull/224
- PR from `mx-chain-es-indexer-go`: (https://github.com/multiversx/mx-chain-es-indexer-go/pull/242)

## Testing procedure
-  Do a smart contract deploy and upgrade, the log event for and deploy and upgrade should contains also the `codeHash`
- Do a `changeOwner` transaction for the deployed contract and check that an event with the identifier `ChangeOwnerAddress` was generated


## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
